### ISSUE TITLE
CW or CCW triangles ignored by the render engine

### ISSUE THREAD

**Issue by ArchlineXP** [ospray/ospray#545]
Is there any way to ignore the CW or CCW triangles during rendering? Some objects are rendered incorrectly, miscolored triangles appear on the surface instead of the color I set. I attached a photo of it.
![OSPRay Render](https://user-images.githubusercontent.com/118745805/203078031-bff6d4b1-7617-4d48-a8bc-c7466bc6731a.png)

**Comment by johguenther** [comment]
No, there is no backface-culling in OSPRay. The artifacts on the chairs indeed look like z-fighting: are there multiple surfaces overlapping at the same spot (two chairs put in each other)? Maybe also the vertex-normals are too extreme (parallel to the surface). Which material is used? Do the artifacts also appear with the SciVis renderer / OBJ material?
